cf-custom-resources/package-lock.json

```diff
@@ -10147,6 +10147,20 @@
         "is-typedarray": "^1.0.0"
       }
     },
+    "node_modules/typescript": {
```

Comment:
where does this dependency come from 🤔

Reply:
After updating npm to the latest version. I won't be able to build because it keeps saying my package-lock is out of sync